README.md

```diff
@@ -39,19 +43,20 @@ In an AJAX POST request, what does the argument of the `.done()` callback - in t
     console.log("success!");
   });
 ```
-### Question 6
-Suppose you had the following form in your HTML file. Use jQuery to write a single line of code that takes whatever the user entered in the textbox and saves it to the variable `user_input`. 
+-once the ajax POST request is finished, return the object that was posted in the ajax call.
 
+### Question 6
+Suppose you had the following form in your HTML file. Use jQuery to write a single line of code that takes whatever the user entered in the textbox and saves it to the variable `user_input`.
 ```
   <form>
     <p>The dress is:</p><input type="text" id="color">
     <input type="submit" value="Submit" id="submit">
   </form>
 ```
+- var user_input = $('submit').val();
```

Comment:
Close, but the submit button doesn't contain what the user typed. For that you need the text input box, which has an ID of color. Also remember an ID needs to have # in front of it. So `var user_input = $('#color').val()`.